modules/configuring-ovs-log-level-permanently.adoc

```diff
@@ -30,7 +29,7 @@ metadata:
 spec:
   config:
     ignition:
-      version: {ign-config-version}
+      version: 3.2.0
```

Comment:
How come we're writing it out now? 

Just curious.

Reply:
@stevsmit the logic
```
:ign-config-version: 3.1.0
ifeval::[{product-version} > 4.6]
:ign-config-version: 3.2.0
endif::[]
```
is a numerical comparison, so when OCP 4.10 was released, numerically it is smaller than 4.6. It's not the first time this kind of version logic has been used in the docs, so there may be more instances across the docs that are similar

Reply:
just seems easier at this point to use 3.2.0, since that is the version used in the docs we currently support. The logic in the configs can be revisited whenever Ignition or another tool has a new release

Reply:
Makes sense. Just wondering!